From 25e91c7c5107eac31e14ebe5630af4547bb74479 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Oliver=20St=C3=A4bler?= <oliver.staebler@bytesatwork.ch>
Date: Wed, 12 Feb 2014 19:53:31 +0100
Subject: [PATCH] bytePANEL: Several small changes for display ET070080DH6

* Change name from EMERGING to ET070080DH6
* Add size to Kconfig
* Adjust timing
---
 arch/arm/mach-omap2/board-am335x-bytepanel.c |    2 +-
 drivers/video/Kconfig                        |    2 +-
 drivers/video/da8xx-fb.c                     |    8 ++++----
 3 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/arch/arm/mach-omap2/board-am335x-bytepanel.c b/arch/arm/mach-omap2/board-am335x-bytepanel.c
index 02a0955..af7fa98 100644
--- a/arch/arm/mach-omap2/board-am335x-bytepanel.c
+++ b/arch/arm/mach-omap2/board-am335x-bytepanel.c
@@ -108,7 +108,7 @@ static struct da8xx_lcdc_platform_data lcdc_pdata = {
 static struct da8xx_lcdc_platform_data lcdc_pdata = {
 	.manu_name		= "Emerging",
 	.controller_data	= &lcd_cfg,
-	.type			= "EMERGING",
+	.type			= "ET070080DH6",
 };
 #elif defined(CONFIG_LCD_ETML101000DH6)
 static struct da8xx_lcdc_platform_data lcdc_pdata = {
diff --git a/drivers/video/Kconfig b/drivers/video/Kconfig
index 7692415..e4cf37d 100644
--- a/drivers/video/Kconfig
+++ b/drivers/video/Kconfig
@@ -2267,7 +2267,7 @@ config LCD_ET050080DM6
 	  If you have an Emerging ET050080DM6 LCD Panel, say Y to enable its driver.
 
 config LCD_ET070080DH6
-	bool "Emerging ET070080DH6 (pRGB)"
+	bool "Emerging ET070080DH6 7\" (pRGB)"
 	select LCD_INVERSE_POL
 	help
 	  If you have an Emerging ET070080DH6 LCD Panel, say Y to enable its driver.
diff --git a/drivers/video/da8xx-fb.c b/drivers/video/da8xx-fb.c
index 23f1e33..bcdfa7a 100644
--- a/drivers/video/da8xx-fb.c
+++ b/drivers/video/da8xx-fb.c
@@ -306,12 +306,12 @@ static struct da8xx_panel known_lcd_panels[] = {
 	},
 	/* Emerging ltt */
 	{
-		.name = "EMERGING",
+		.name = "ET070080DH6",
 		.width = 800,
 		.height = 480,
-		.hfp = 2,
-		.hbp = 216,
-		.hsw = 30,
+		.hfp = 1,
+		.hbp = 215,
+		.hsw = 40,
 		.vfp = 8,
 		.vbp = 35,
 		.vsw = 2,
-- 
1.7.10.4

